src/ucp/proto/proto_am.inl

```diff
@@ -476,6 +476,11 @@ ucp_proto_ssend_ack_request_alloc(ucp_worker_h worker, uintptr_t ep_ptr)
 
     req->flags              = 0;
     req->send.ep            = ucp_worker_get_ep_by_ptr(worker, ep_ptr);
+    if (req->send.ep == NULL) {
+        ucp_request_put(req);
+        return NULL;
```

Comment:
need to handle NULL in the caller to ucp_proto_ssend_ack_request_alloc

Reply:
NULL is currently handled with ucs_fatal in the calling functions.
you mean it should be handled differently?

Reply:
yes, instead of fatal need to cancel the operation

Reply:
cancel it even though it was just allocated with ucp_request_get?

Reply:
need to avoid sending sync_ack